java/ql/test/library-tests/frameworks/android/uri/test.ql

```diff
@@ -0,0 +1,53 @@
+import java
+import semmle.code.java.dataflow.DataFlow
+import semmle.code.java.dataflow.ExternalFlow
+import semmle.code.java.dataflow.TaintTracking
+import TestUtilities.InlineExpectationsTest
```

Comment:
`java/ql/test/TestUtilities/InlineFlowTest.qll` can be used imported of the boilerplate code.

Reply:
Done